plugins/plugins-available/conf/lib/Monitoring/Config.pm

```diff
@@ -292,11 +291,16 @@ sub commit {
         local $ENV{THRUK_BACKEND_ID}   = $c->stash->{'param_backend'};
         local $ENV{THRUK_BACKEND_NAME} = $backend_name;
         local $SIG{CHLD}               = 'DEFAULT';
-        system($c->config->{'Thruk::Plugin::ConfigTool'}->{'post_obj_save_cmd'}, 'post', $filesroot);
-        if($? == -1) {
-            Thruk::Utils::set_message( $c, 'fail_message', 'post save hook failed: '.$?.': '.$! );
+        my $cmd = $c->config->{'Thruk::Plugin::ConfigTool'}->{'post_obj_save_cmd'}." post '".$filesroot."' 2>&1";
+        my $out = `$cmd`;
+        my $rc  = $?>>8;
+        $c->log->info("post save hook: '" . $cmd . "', rc: " . $rc);
+        if($rc != 0) {
+            $c->log->info('post save hook out: '.$out);
+            Thruk::Utils::set_message( $c, 'fail_message', "Save canceled by post_obj_save_cmd hook!\n".$out );
```

Comment:
This message is wrong, the save has already happenend at this stage. We can only inform the user that the post hook has failed.